inc/cppxaml/XamlProperty.h

```diff
@@ -0,0 +1,117 @@
+#pragma once
+#include <type_traits>
+
+namespace winrt {
+    template<typename T, typename I>
+    T try_as(I* i) {

```

Comment:
do you need this? winrt::com_ptr has try_as

Reply:
it does - however that works for the interface type, not the implementation type... I added this because I wasn't sure how to go from the implementation type to the interface IInspectable... there might be an easier way :D